adr/ADR-44.md

```diff
@@ -109,13 +109,11 @@ API level only needs to be incremented once and not for every new feature.
 We'll store current server and asset related information in the existing `metadata` field allowing us to expand this in 
 time, today we propose the following:
 
-| Name                             | Description                                               |
-|----------------------------------|-----------------------------------------------------------|
-| `_nats.server.version`           | The current server version hosting an asset               |
-| `_nats.server.api_level`         | The current server API level hosting an asset             |
-| `_nats.server.require.api_level` | The required API level to start an asset                  |
-| `_nats.created.server.version`   | The version of the server that first created this asset   |
-| `_nats.created.server.api_level` | The API level of the server that first created this asset |
+| Name              | Description                                   |
+|-------------------|-----------------------------------------------|
+| `_nats.ver`       | The current server version hosting an asset   |
+| `_nats.level`     | The current server API level hosting an asset |
+| `_nats.req.level` | The required API level to start an asset      |
```

Comment:
I tried to lower the impact without making it weird - like I am sure we could combine level and version in one entry but as discussed we need to solve the actual problem of info/config split rather than band aids.

So its smaller and fewer bytes and some fields removed but still obvious how it works.

Reply:
Can it just be one key like `_nats.ver` and the value is multiple values in an array that follows that ordering?

Reply:
It can but as above we want it to be meaningful and obvious to users what this is. Not needlessly obfuscate the design when that does not solve the problem - we are going to be adding more stuff to state and config. So this significantly reduce the byte size but balancing UX vs the byte count need

So we will try add a split to info/state APIs for 2.11 to improve the underlying cause of this

Reply:
A design based on key ordering having magical value will get us the same place we are with the reply subject for stream msg metadata.

Reply:
This is only on the wire, once in the client can be expanding out to separate keys / values no?

Reply:
The values are UX. it’s not an invisible feature that we hide its publically visible and answering “what is this?” On slack 500 times over is much more expensive than bytes on the wire.

Reply:
I agree here with @ripienaar .
The API has to be clear and well defined.
If it's not, it causes bugs in clients and misunderstandings for users.
Especially as users will have access to it even if client do nothing about it.

Reply:
What about encoding the `_nats` entry as JSON, that would allow some of the keys to compact?

expanded: 63
```json
 {"_nats.ver":"2.11.20","_nats.level":"0","_nats.req.level":"0"}
```
compact: 52
```json
 {"_nats":"{\"ver\":\"2.12.1\",\"api\":1,\"min\":0}"}
```

We could also remove 8 more bytes, if we had a struct that managed these values outside of metadata, since these are system values....

Reply:
seems pretty awful and additional overhead with json encoding.

They could be struct keys but I avoided that since I think we'll add more over time and want a bit of flexibility here without churning all the clients and data structs

Reply:
standalone:
```json
 {"ver":{"srv":"2.12.1","api":1,"min":0}}
```